### ISSUE TITLE
Does not display * for default model

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#3006]
Syntax + output:
~~~
julien-macbook:~/code/sct $ sct_deepseg -list-tasks

--
Spinal Cord Toolbox (git-master-66f2246ffd904ee0c7345c7d309c92fc6edba850)


--
Spinal Cord Toolbox (git-master-66f2246ffd904ee0c7345c7d309c92fc6edba850)

TASK                DESCRIPTION                                       MODELS
--------------------------------------------------------------------------------
seg_sc_t2star       Cord segmentation on T2*-weighted contrast.       t2star_sc
seg_mice_sc         Cord segmentation on mouse MRI.                   mice_uqueensland_sc
seg_mice_gm         Gray matter segmentation on mouse MRI.            mice_uqueensland_gm
seg_tumor_t2        Cord tumor segmentation on T2-weighted contrast.  findcord_tumor, t2_tumor

Legend: installed | not installed | default: [*]
~~~

Expected behaviour: a [*] should appear next to `seg_sc_t2star` because it is a default model. 

A regression might have been introduced at some point (it used to work).

**Comment by jcohenadad** [comment]
Nevermind, the [*] should only appear for the following command:
~~~
sct_deepseg -list-models
~~~

it should not appear for the task, and therefore the Legend is misleading and will be removed in #2996

**Comment by jcohenadad** [comment]
Fixed by #2996
